packages/ui5-application-writer/src/index.ts

```diff
@@ -47,15 +47,19 @@ async function generate(basePath: string, ui5AppConfig: Ui5App, fs?: Editor): Pr
     });
     ui5Config.addFioriToolsAppReloadMiddleware();
 
+    // preload ui5-local.yaml
+    const ui5DefaultLibs = ['sap.m', 'sap.ushell', 'sap.ui.core'];
+    const ui5Libs = ui5App.ui5.ui5Libs as string[];
+    ui5DefaultLibs.forEach((lib) => {
+        if (!ui5Libs.includes(lib)) {
+            ui5Libs.push(lib);
+        }
+    });
+
```

Comment:
`sap.ushell` should added in by the FE and FF writers as they add they `flpSandbox.html` files which needs `sap.ushell`

`sap.m` &&  `sap.ui.core` can be added in ui5-application-writer by default. 
Which seems to be already covered here https://github.com/SAP/open-ux-tools/blob/ebb77d926a837e56741e8ff378a7a8d0466e7f7e/packages/ui5-application-writer/src/data/defaults.ts#L84

Reply:
Addressed in [a43bc3c](https://github.com/SAP/open-ux-tools/pull/1948/commits/a43bc3cd3d90fc3f9565fae57ca54c95b007f80b) and [74e857c](https://github.com/SAP/open-ux-tools/pull/1948/commits/74e857ca2c3c9c1be08866ed40ddb12f8b409a73).
Thanks @devinea